src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -58,7 +58,7 @@ public KafkaRepository(final ZooKeeperHolder zkFactory, final KafkaFactory kafka
                     .map(Topic::new)
                     .collect(Collectors.toList());
         } catch (Exception e) {
```

Comment:
Exception is too generic for unavailable. If there is a network problem I would agree with this. But if it's just a generic error, then it seems that 500 fits best.